<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-811Q52AV" failures="1" tests="5" name="com.framework.runner.TestRunner" time="8.900" errors="0" timestamp="2021-12-27T20:33:20 SGT" skipped="0">
  <testcase classname="com.framework.runner.TestRunner" name="runScenario" time="0.451"/>
  <system-out/>
  <testcase classname="com.framework.runner.TestRunner" name="runScenario" time="1.403"/>
  <system-out/>
  <testcase classname="com.framework.runner.TestRunner" name="runScenario" time="6.817"/>
  <system-out/>
  <testcase classname="com.framework.runner.TestRunner" name="runScenario" time="0.110">
    <failure message="expected [2001] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [2001] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.framework.StepDefinition.API_Steps.i_should_get_the_response_status_as(API_Steps.java:33)
at âœ½.I should get the response status as 2001(file:///C:/Users/udit/Desktop/workspace/Assignment/src/test/resources/features/API_TestCase.feature:28)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.framework.runner.TestRunner" name="runScenario" time="0.119"/>
  <system-out/>
</testsuite> <!-- com.framework.runner.TestRunner -->
